conf/routes

```diff
@@ -20,7 +20,7 @@ OPTIONS        /paypal/auth                     controllers.PaypalController.aut
 POST           /paypal/capture                  controllers.PaypalController.capturePayment
 GET            /paypal/:countryGroup/execute    controllers.PaypalController.executePayment(countryGroup: CountryGroup, paymentId, token, PayerID, CMP: Option[String], INTCMP: Option[String], refererPageviewId: Option[String], refererUrl: Option[String],pvid: Option[String], bid: Option[String], ophanVisitId: Option[String], componentId: Option[String], componentType: Option[ComponentType], source: Option[AcquisitionSource], refererAbTest: Option[AbTest], nativeAbTests: Option[Set[AbTest]], supportRedirect: Option[Boolean])
 POST           /paypal/hook                     controllers.PaypalController.hook
-POST           /:countryGroup/update-metadata   controllers.PaypalController.updateMetadata(countryGroup:CountryGroup)
+POST           /:countryGroup/update-marketing  controllers.IdentityController.updateMarketingPreferences(countryGroup:CountryGroup)
```

Comment:
Keep both routes for now to avoid breaking cached JS

Reply:
Done